db/migrate/20231022224833_update_custom_field_data.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class UpdateCustomFieldData < ActiveRecord::Migration[7.0]
+  def up
```

Comment:
I think since this is a complex migration we should add a migration spec for the PR too (and remove it before merge). Just so I understand, this is making it so the value for the automation field would be `user_field_blah` instead of just e.g. `Title`?

Reply:
Oh I see some were using the name and some were using the value...I think a spec will make this clearer and add more confidence